locations/models.py

```diff
@@ -77,8 +77,7 @@ def get_context(self, request):
                         if d <= self.max_dist_km:
                             locationsnearme.append(page)
         context['locationsnearme'] = locationsnearme
-        locationtagindexset=LocationTagIndexPage.objects.live().child_of(self)
-        context['tagspage']=locationtagindexset[0]
+        context['tagspage'] = LocationTagIndexPage.objects.live().child_of(self).first()
```

Comment:
maybe a comment would be good: `# the first tag index page which is a child of this page`